bridge/contracts/nftbridge/NFTBridge.sol

```diff
@@ -46,8 +46,8 @@ contract NFTBridge is
   IERC1820Registry internal constant ERC1820 =
       IERC1820Registry(0x1820a4B7618BdE71Dce8cdc73aAB6C95905faD24);
 
-  address internal federation;
-  uint256 internal feePercentage;
+  address payable internal federation;
```

Comment:
how did the federation manage to receive ether before being declared as payable?
maybe the question is - is this absolutely necessary or does it only improve readability? (just to understand, not requesting any changes)

Reply:
before like in the other bridge the federation didn't receive ether directly it was always the token ERC20 that was being crossed